test/e2e_node/node_shutdown_linux_test.go

```diff
@@ -343,11 +343,12 @@ var _ = SIGDescribe("GracefulNodeShutdown [Serial] [NodeFeature:GracefulNodeShut
 			// Wait a few seconds to ensure dbus is restarted...
 			time.Sleep(5 * time.Second)
 
```

Comment:
```suggestion
```

There is no need to wait here since retries have been added

Reply:
Updated.